src/components/actionMenu/actionMenu.tsx

```diff
@@ -43,6 +43,7 @@ const ActionMenu = <T,>({ item, actions }: ActionMenuProps<T>) => {
             onClick={() => {
               a.onClick(item);
             }}
+            isDisabled={a.disabled}
```

Comment:
Here. If I change this to `disabled` I end up with download buttons always enabled :/

Reply:
i'm not sure what that means, but this is part of the solution, leave as is